MantidQt/CustomInterfaces/src/Muon/MuonAnalysisResultTableTab.cpp

```diff
@@ -497,40 +497,47 @@ void MuonAnalysisResultTableTab::populateLogsAndValues(
     // Get log information
     std::string wsName = fittedWsList[i].toStdString();
     auto ws = retrieveWSChecked<ExperimentInfo>(wsName + WORKSPACE_POSTFIX);
-
-    Mantid::Kernel::DateAndTime start = ws->run().startTime();
-    Mantid::Kernel::DateAndTime end = ws->run().endTime();
-
     const std::vector<Property *> &logData = ws->run().getLogData();
-
+    const TimeSeriesProperty<bool> *runningLog; // defined here to keep the
```

Comment:
We should always initialize variables to some known value. This is not guaranteed to be a nullptr so we should make it one:
```
const TimeSeriesProperty<bool> *runningLog(nullptr);
```